pkg/scheduler/core/generic_scheduler.go

```diff
@@ -336,6 +341,20 @@ func (g *genericScheduler) getLowerPriorityNominatedPods(pod *v1.Pod, nodeName s
 	return lowerPriorityPods
 }
 
+// numFeasibleNodesToFind returns the number of feasible nodes that once found, the scheduler stops
+// its search for more feasible nodes.
+func (g *genericScheduler) numFeasibleNodesToFind(allNodes int32) int32 {
+	if allNodes < minFeasibleNodesToFind || g.percentageOfNodesToScore <= 0 ||
```

Comment:
can we check range of `percentageOfNodesToScore` in `apiserver` and `kube-schedule.cmd`?